<launch>
	<node pkg="auto_pilot"  type="GPS_Publish.py" respawn="true" name="GPS"  />
	<node pkg="auto_pilot"  type="Compass_from_SmartPhone.py" respawn="true" name="Compass"/> 
	<node pkg="auto_pilot"  type="Global_path.py" respawn="true" name="Path_Planner" output = "screen"/>
	<node pkg="auto_pilot"  type="Thruster_Controller.py" respawn="true" name="Thruster_Controller"/>

	<node pkg="rosserial_python" type="serial_node.py" name="arduino" respawn="true" args="/dev/ttyACM0"></node>
	<!--<node name="rqt" pkg="rqt_gui" type="rqt_gui" />-->

	<rosparam command="load" file="$(find auto_pilot)/params/AutoPilot_params.yaml"/>

</launch>
